src/components/radio-group/radio-group.sandbox.html

```diff
@@ -1,6 +1,8 @@
 <div>
     <h3>Add a margin below the radio button group</h3>
-    <p>From <a href="https://jira.dti.ulaval.ca/browse/MODUL-135">MODUL-135</a></p>
+    <p>From <a href="https://jira.dti.ulaval.ca/browse/MODUL-135"
+           target="_blank">MODUL-135</a> et <a href="https://jira.dti.ulaval.ca/browse/MODUL-798"
```

Comment:
Est-ce qu'on veut vraiment mettre des liens de jira dans modUL ou ça serait mieux vers le PR qui explique la correction? Une personne externe ne comprendra pas le problème.
 Et changer le "et" pour "and" ;)

Reply:
C'est fait